autoscale/src/main/java/com/sequenceiq/periscope/controller/DistroXAutoScaleClusterV1Controller.java

```diff
@@ -144,6 +144,7 @@ private DistroXAutoscaleClusterResponse updateClusterAutoScaleConfig(Cluster clu
         }
         alertValidator.validateScheduleWithStopStart(cluster, autoscaleClusterRequest);
 
+        autoscaleClusterRequest.setUseStopStartMechanism(alertValidator.enableStopStartBasedOnEntitlement(cluster));
```

Comment:
This call needs to only enable the mechanism if there are load based policies. We should never enable the stopstart mechanism for schedule based policies.

Also the call is a little strange, since it sends it in the request - and also updates the DB directly via the alertValidator change (separate comment on that).

Reply:
Done.
Shifted the setting of stopStart above it and doing a check of schedule based mechanism before and after the setting of value.

Reply:
I changed the DB value because this function was also changing it:- validateStopStartEntitlementAndDisableIfNotEntitled.
So, I thought it would be required.

Reply:
That code also throws an error if the cluster is not entitled. It's likely doing some error handling from previous clusters where we may have enabled it. This is not the place to make this change.

Reply:
This check also doesn't bother with schedule or load based policies.